src/test/pmem_memcpy/README

```diff
@@ -0,0 +1,32 @@
+Linux NVM Library
+
+This is src/test/pmem_memcpy/README.
+
+This directory contains a unit test for pmem_memcpy().
+
+The program in pmem_memcpy.c takes a file name as an argument.
+
+Initially the program maps the file. It then does an anonymous mapping to
+be used as the source address in the subsequent calls to pmem_memcpy.
+There are several test cases this program runs:
+	src addr > dest addr
+	src addr < dest addr
+<<<<<<< HEAD
+<<<<<<< HEAD
+
+In the case of memcpy() overlapping addresses is not supported.
+=======
+=======
+>>>>>>> c911599f73b62bc85aac6def9e91086f6bf2e210
+	With these sub tests:
+		aligned src and dest
+		unaligned src, aligned dest
+		unaligned src, unaligned dest
+<<<<<<< HEAD
+>>>>>>> 2077a2e... code: Add support for pmem non-temporal memcpy, memmove and memset
+=======
+>>>>>>> c911599f73b62bc85aac6def9e91086f6bf2e210
```

Comment:
Sorry, I will fix. I thought I got all of these. Isn’t cstyle supposed to catch these kind of issues? I ran it before I integrated.

From: Lukasz Dorau [mailto:notifications@github.com]
Sent: Friday, January 9, 2015 1:46 AM
To: pmem/nvml
Cc: Jelinek, Sarah
Subject: Re: [nvml] code: Add support for pmem non-temporal memcpy, memmove and memset (#98)


In src/test/pmem_memcpy/README<https://github.com/pmem/nvml/pull/98#discussion-diff-22705794>:

> +     src addr < dest addr

> +<<<<<<< HEAD

> +<<<<<<< HEAD

> +

> +In the case of memcpy() overlapping addresses is not supported.

> +=======

> +=======

> +>>>>>>> c911599f73b62bc85aac6def9e91086f6bf2e210

> +     With these sub tests:

> +            aligned src and dest

> +            unaligned src, aligned dest

> +            unaligned src, unaligned dest

> +<<<<<<< HEAD

> +>>>>>>> 2077a2e... code: Add support for pmem non-temporal memcpy, memmove and memset

> +=======

> +>>>>>>> c911599f73b62bc85aac6def9e91086f6bf2e210

Looks like unresolved merge conflict...

—
Reply to this email directly or view it on GitHub<https://github.com/pmem/nvml/pull/98/files#r22705794>.